src/main/java/dateBases/databasesutils/MySqlConnector.java

```diff
@@ -0,0 +1,48 @@
+package dateBases.databasesutils;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.apache.log4j.Logger;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+@Getter
+@NoArgsConstructor
+public class MySqlConnector {
+    private static Properties properties = new Properties();
+    private static String propertiesPath = "C:\\Users\\Sergey\\Desktop\\api-samples-master\\HomeWork13\\src\\main\\" +
```

Comment:
Use relative path

Reply:
![image](https://user-images.githubusercontent.com/41234925/56022287-fc4a4b80-5d13-11e9-9c09-588b6be1ef8c.png)
Doesn't work with relative path

Reply:
It does. But you should change your code. 
https://www.baeldung.com/reading-file-in-java